shfmt/entrypoint.sh

```diff
@@ -5,12 +5,16 @@ set -euo pipefail
 # shellcheck disable=SC1091
 source /lib.sh
 
+if [ -z "${SHFMT_ARGS:-}" ]; then
+	SHFMT_ARGS=""
+fi
+
 fix() {
-	shfmt -s -w .
+	shfmt -s -w ${SHFMT_ARGS} .
```

Comment:
What do you think of in-lining it directly? We could avoid the `[ -z ]` test with that, and it's an idiom I'm used to use on `bash`.

```suggestion
	shfmt -s -w ${SHFMT_ARGS:-} .
```

Reply:
Yes good suggestions, I've committed it.